.travis.yml

```diff
@@ -1,7 +1,17 @@
 language: node_js
-install:
-  - yarn
+addons:
+  apt:
+    packages:
+      - libgconf-2-4
+install: yarn install --frozen-lockfile
+cache:
+  yarn: true
+  directories:
+    - ~/.cache
+env:
+  - NODE_ENV=ci
 script:
   - yarn test
```

Comment:
Travis should also run a linting step before `testing`

Reply:
We have https://app.asana.com/0/1136962714401929/1146342193058673/f is to fix our linting errors, and then also add linting before test as you're recommending, thanks, but I don't want to wait to get this in before hooking up CI/CD as ensuring the Form Engine API doesn't get inadvertently borked is top priority for all this to me, and I'd like to get that defense in if possible asap.